python/mujoco/usd/exporter.py

```diff
@@ -15,13 +15,16 @@
 """USD exporter."""
 
 import os
-from typing import List, Optional, Union
+from typing import List, Optional, Tuple, Union
 
 import mujoco
 import mujoco.usd.camera as camera_module
 import mujoco.usd.lights as light_module
 import mujoco.usd.objects as object_module
 import mujoco.usd.shapes as shapes_module
+from mujoco import _enums
```

Comment:
You shouldn't be importing these internal modules. Everything from these modules is exposed directly from the mujoco module.